### PR TITLE
[WALLPAPERS] Restructuring

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
Respect the original directory structure.
JIRA issue: [CORE-19617](https://jira.reactos.org/browse/CORE-19617)

## Proposed changes

- Make `Angelus`, `Angelus/ReactOS`, and `Mizu` folders.
- Move image files.
- Add `LICENSE.txt` and `suggestion.txt` files.
- Revive `modules/wallpapers/Angelus/ReadMe.txt`.

## TODO

- [ ] Check by @HBelusca

### CODE REVIEW

File: modules/wallpapers/LICENSE.txt [hunk 1]
```diff
@@ -67,3 +67,4 @@ History:
 
 2005-02-25 Klemens Friedl
 2005-05-31 Klemens Friedl
+2024-05-30 Katayama Hirofumi MZ
```
Comment (binarymaster) [inline modules/wallpapers/LICENSE.txt:70]: (you didn't modified the wallpapers)

Comment (katahiromz) [reply modules/wallpapers/LICENSE.txt]: Killed.

File: modules/wallpapers/Angelus/ReactOS/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/Angelus.ReactOS7.Dark_16-9.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/Angelus.ReactOS7.Dark_4-3.jpg DESTINATION reactos/Web/Wallpaper FOR all)
```
[the remark this answers is not in the document]
Comment (katahiromz) [reply modules/wallpapers/Angelus/ReactOS/CMakeLists.txt:2]: image
Unable for me.

File: modules/wallpapers/Angelus/ReadMe.txt [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+ReactOS Sunset
+--------------
+The photo was taken in a colorful sunset where the author lives. We can see the new Earth's satellite ;-)
+
+Sea Series
+----------
+1 (Golden Sunset): Beautiful sunset in the Atlantic ocean.
+2: Photo taken at Praia do Vilar (Vilar's Beach) in the Barbanza region.
+3 (Red Sunset): Photo taken from Carril, a beautiful village at the port town of Vilagarc�a de Arousa.
```
Comment (katahiromz) [inline modules/wallpapers/Angelus/ReadMe.txt:46]: Mojibake.

Comment (whindsaks) [reply modules/wallpapers/Angelus/ReadMe.txt:46]: Make the file UTF-8? [Vilagarcía de Arousa](https://en.wikipedia.org/wiki/Vilagarc%C3%ADa_de_Arousa)

Comment (katahiromz) [reply modules/wallpapers/Angelus/ReadMe.txt:46]:
```suggestion
3 (Red Sunset): Photo taken from Carril, a beautiful village at the port town of Vilagarcía de Arousa.
```

**Comment by HBelusca** [comment]
Just to point stuff, you can indeed keep the original directory structure (including in the reactos/wallpapers/data sub-project, see https://github.com/reactos/wallpapers/issues/2 ), by remembering that previously in master we were able to maintain such structure, but when constructing the ISOs with the wallpapers enabled we could flatten this structure.... _if I recall correctly_.
Since we were using our build system there, we could have each subdirectory having its own CMakeLists.txt file, having the `add_cd_file()` command to add whatever file to a given place in the ISO, independently on how it was stored in our source tree.
I guess you could reproduce that in the subproject when building your zip archive, if somehow there is a script that iterates through all the files (and recursively goes into the subdirectories), and for each encountered, add them into the archive at the proper place (all at the same level, or whatever you choose).

**Comment by katahiromz** [comment]
> having the add_cd_file() command

I cannot write `CMake` scripts. No skill data.

**Comment by katahiromz** [comment]
https://github.com/reactos/wallpapers/pull/3

**Comment by katahiromz** [comment]
integration-ok
Integration is OK.

**Review by learn-more — APPROVED** [review]

**Pull request merged**